src/cantools/database/can/formats/arxml/system_loader.py

```diff
@@ -244,15 +269,42 @@ def _load_buses(self, package_list):
 
                     buses.append(Bus(name=name,
                                      comment=comments,
-                                     autosar_specifics=autosar_specifics,
+                                     autosar_specifics=AutosarBusSpecifics(cluster_name=cluster_name),
                                      baudrate=baudrate,
                                      fd_baudrate=fd_baudrate))
                 else: # AUTOSAR 3
-                    name = \
+                    cluster_name = \
                         self._get_unique_arxml_child(can_cluster,
                                                      'SHORT-NAME').text
                     comments = self._load_comments(can_cluster)
 
+                    # physical channels
+                    # [AUTOSAR SystemTemplate - Table 2.6: PhysicalChannel]
+                    # ...
+                    # Note:  ... Bus systems like CAN and LIN only have exactly one PhysicalChannel.
+                    physical_channels = \
+                        self._get_arxml_children(can_cluster,
+                                                    [
+                                                        'PHYSICAL-CHANNELS',
+                                                        '*&PHYSICAL-CHANNEL'
+                                                    ])
+                    
+                    if physical_channels is None or len(physical_channels) == 0:
+                        # WTH?
```

Comment:
probably a leftover, otherwise let's expand the comment

Reply:
It was used the same in the rest of the library. Added just to keep consistency. Should I change it everywhere?